.github/workflows/release.yml

```diff
@@ -0,0 +1,34 @@
+name: Release
+on:
+  push:
+    branches:
+      - master
+jobs:
+  release:
+    name: Build and Release
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      issues: write
+      pull-requests: write
+      id-token: write
+    steps:
+      - name: Checkout
+        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11
+        with:
+          fetch-depth: 0
+      - name: Setup Node.js
+        uses: actions/setup-node@b39b52d1213e96004bfcb1c61a8a6fa8ab84f3e8
+        with:
+          node-version: 21.x
```

Comment:
why it is 21?

Reply:
Because it's the `latest` and it's stable. Do you have any concerns with it? 🤔

Reply:
@jim-deriv I suggest sticking to Node 20 LTS as of now v21 is not getting any updates other than security updates which are going to be ended in a month

ref: https://endoflife.date/nodejs